dsc/tests/dsc_export.tests.ps1

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+Describe 'resource export tests' {
+    
+    It 'Export can be called on individual resource' {
+
+        $processes = dsc resource export -r Microsoft/Process
+        $LASTEXITCODE | Should -Be 0
+        $processes.count | Should -BeGreaterThan 1
+    }
+
+    It 'Export can be called on a configuration' {
+
+        $json = @'
+            $schema: https://schemas.microsoft.com/dsc/2023/03/configuration.schema.json
+            resources:
+            - name: Processes
```

Comment:
It seems that there isn't a good case where you would export the same resource twice, we should add some validation in `dsc` to emit an error if the same resource is specified more than once for export